docs/community/working-group-definitions.md

```diff
@@ -91,6 +91,7 @@ These topics include (but are not limited to):
 * @SeeminglyScience
 * @sdwheeler
 * @kilasuit
+* @FriedrichWeinmann
```

Comment:
```suggestion
* @FriedrichWeinmann
* @StevenBucher98 
```

Reply:
I'm not included yet in the list either 😄

Reply:
Thanks for the catch - updated!